frontend/dispatcher.lua

```diff
@@ -992,10 +992,22 @@ function Dispatcher:isActionEnabled(action)
     return not disabled
 end
 
-function Dispatcher:_showAsMenu(settings, gesture)
+function Dispatcher:_showAsMenu(settings, exec_props)
     local display_list = Dispatcher:getDisplayList(settings)
     local quickmenu
     local buttons = {}
+    if exec_props and exec_props.qm_show then
+        table.insert(buttons, {{
+            text = _("Execute all"),
+            align = "left",
+            font_face = "smallinfofont",
+            font_size = 22,
+            callback = function()
+                UIManager:close(quickmenu)
+                Dispatcher:execute(settings, { qm_show = false })
```

Comment:
Missing a nil or false "gesture" as the 2nd argument ?
(I guess it would look odd if we put "gesture" inside the exec_props table ?)

Reply:
> (I guess it would look odd if we put "gesture" inside the exec_props table ?

I think it'll simplify a little bit.
@yparitcher can you please review.